.postcssrc

```diff
@@ -3,7 +3,10 @@
   "plugins": {
     "postcss-custom-properties": {
       "preserve": false,
-      "importFrom": "src/styles/variables.css"
-    }
+      "importFrom": [
+        "src/styles/variables.css",
+        ]
+    },
+    autoprefixer: { browsers: ['last 2 versions', 'iOS >= 8'] }
```

Comment:
Let autoprefixer get the browserlist variable from package.json. It's already set and is synced with the other repos.